calico-cloud/operations/monitor/prometheus/byo-prometheus.mdx

```diff
@@ -272,24 +272,6 @@ For all Prometheus Felix configuration values, see [Felix Prometheus](../../../r
 
 **Create a service to expose Felix metrics**
 
-```bash
```

Comment:
Technically, for cloud this should be merged once they integrate v3.22.0-1.0, is that right? Because that is the first operator release that will have the changes for cloud users. 
@ctauchen What is the best way to make sure we merge the diffs in this file later?

Reply:
Yes, thank you for correcting me.